### PR TITLE
Increase default runner to pulumi-ubuntu-8core

### PR DESCRIPTION
**Pull request by rquitales**
To help unblock all CI jobs.

### CODE REVIEW

File: .ci-mgmt.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ team: ecosystem
 goBuildParallelism: 2
 javaGenVersion: "v0.8.0"
 runner:
+  default: pulumi-ubuntu-8core
   prerequisites: pulumi-ubuntu-8core
   buildSdk: pulumi-ubuntu-8core
   publish: pulumi-ubuntu-8core
```
Comment (iwahbe) [inline .ci-mgmt.yaml:34]: Nit: We can comment these out if we want, since they are overriding the default *and* the same as the default.

**Review by iwahbe — APPROVED** [review]

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
I think most of these aren't really necessary.

Most of these were addressed via logic in the ci-mgmt templates (the templates decide which runner to assign based on the config field set), and passed to pulumi-azure's Workflows via [PRs like this one](https://github.com/pulumi/pulumi-azure/commit/f2928e097779e9a0f039665b26955faf60520032).

See also changes applied [here](https://github.com/pulumi/pulumi-azure/blob/master/.github/workflows/release.yml#L222).

I think we can close this out.

**Comment by iwahbe** [comment]
@rquitales Can we close this?

**Comment by rquitales** [comment]
Stale PR.

**Pull request closed without merging**